client/src/plugins/error-tracking/ErrorTracking.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import debug from 'debug';
+
+import { PureComponent } from 'react';
+
+import * as Sentry from '@sentry/browser';
+
+import Metadata from '../../util/Metadata';
+
+import Flags, { OVERRIDE_SENTRY_DSN } from '../../util/Flags';
```

Comment:
Why not call this one `SENTRY_DSN`? We do the same with `UPDATE_SERVER_URL`, it is not called `OVERRIDE_UPDATE...`, too.

Reply:
Done via 53a8de1a2db68f6a48726b9ca1c12372cece69ec